UPDATING.md

```diff
@@ -70,6 +70,11 @@ https://developers.google.com/style/inclusive-documentation
 
 -->
 
+### Deprecated PodDefaults and in airflow.kubernetes.pod_generator
```

Comment:
'and in' ?

Reply:
@kaxil fixed. Trying to keep up with your comments :)